support-frontend/assets/pages/supporter-plus-landing/components/countdown.tsx

```diff
@@ -106,15 +113,17 @@ export default function Countdown({
 			const isActive =
 				campaign.countdownStartInMillis < now &&
 				campaign.countdownDeadlineInMillis > now;
-			setShowCountdown(isActive);
+			setShow(isActive);
```

Comment:
I think `setShowCountdown` might be a clearer name for this function, as it directly reflects its purpose. That said, if there’s a specific reason for the renaming that I might have missed, so feel free to ignore

Reply:
I've changed those, thanks!